### ISSUE TITLE
learning rate for single task fine tune

### ISSUE THREAD

**Issue by fyubang** [namisan/mt-dnn#70]
Thanks for your work.
I have a question:
What is the optimizer and the learning rate for fine-tuning the single task after mt-dnn?

**Comment by namisan** [comment]
Please refer our paper, I use the same setting.
